app/services/foreman_openscap/client_config/puppet.rb

```diff
@@ -21,13 +21,17 @@ def inline_help
         }
       end
 
+      def collection_method
+        :puppetclasses
```

Comment:
This will not work, HG, nor Host exposes puppetclasses and all_puppetclasses.
It has moved to `(host || hg).puppet` (what is a Facet for puppet)

Reply:
couldn't we let the config to extract the collection? it would give us more flexibility and we could call the object.puppet.collection :thinking:

Reply:
We could do that as an emergency solution, it should work with https://github.com/theforeman/foreman_puppet/pull/180